utils/componentUtils.py

```diff
@@ -5,49 +5,50 @@
 import components
 import typing
 
-def testComponentCompatibility(robot, component_type):
-    """
-    takes a robot and a component_type to check
-    If the component is not compatibile with the robot type, it will attempt to create basic bindings and
-    disable the on_enable and execute() methods of the compeont.
-    """
-    # Iterate over variables with type annotations
-    if not hasattr(component_type, "compatString"):
-        robot.logger.warn("%s has no compatString set. Assuming compatible", component_type)
-        return
-
-    if robot.map.configMapper.checkCompatibilty(component_type.compatString):
-        return
-
-    robot.logger.warn("%s is not compatible. Disabling", component_type)
-
-    for n, inject_type in typing.get_type_hints(component_type).items():
-        # If the variable is private ignore it
-        if n.startswith("_"):
+def testComponentListCompatibility(robot, componentList):
+    print(componentList)
```

Comment:
Need comments from old testComponentCapatablity so we know what the method is doing